.github/dependabot.yml

```diff
@@ -8,11 +8,11 @@ updates:
       # Check for updates to GitHub Actions every week
       interval: "weekly"
     commit-message:
-      prefix: "chore"
+      prefix: "build"
```

Comment:
In my mind, for github actions updates, it's fine to have them as "chore" or "ci"? We only care about incrementing the patch version when the code dependencies are updated?